packages/map-template/src/components/Map/Map.jsx

```diff
@@ -89,15 +91,25 @@ function Map({ onLocationClick, onVenueChangedOnMap }) {
     }, [venueName, venues, startZoomLevel]); // eslint-disable-line react-hooks/exhaustive-deps
     // We ignore eslint warnings about missing dependencies because mapsIndoorsInstance should never change runtime anyway.
 
-    /*
+    /**
```

Comment:
This is a function call, not a function definition, so it should not have a JSDoc block.

Reply:
✅